taxonomies/packaging_materials.txt

```diff
@@ -187,6 +191,7 @@ nl_be:Acrylonitril-butadieen-styreen, ABS
 <en:Plastic
 <en:Other plastics
 en:PLA - Polylactic acid, Polylactic acid
+de:PLA, Polylactide, PLA - Polylactide
```

Comment:
by convention we currently put both the acronym and the full name as the main name

```suggestion
de:PLA - Polylactide, PLA, Polylactide
```

Reply:
Okay, thanks